core/src/main/java/uk/gov/gchq/koryphe/iterable/ChainedIterator.java

```diff
@@ -16,9 +16,12 @@
 
 package uk.gov.gchq.koryphe.iterable;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
I would recommend against this unless there's no other easy way to do this with external libraries.